Formula/linux-pam.rb

```diff
@@ -0,0 +1,45 @@
+class LinuxPam < Formula
+  desc "Pluggable Authentication Modules for Linux"
+  homepage "http://www.linux-pam.org"
+  url "https://github.com/linux-pam/linux-pam/releases/download/v1.5.1/Linux-PAM-1.5.1.tar.xz"
+  sha256 "201d40730b1135b1b3cdea09f2c28ac634d73181ccd0172ceddee3649c5792fc"
+  license any_of: ["BSD-3-Clause", "GPL-1.0-only"]
+  head "https://github.com/linux-pam/linux-pam.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "berkeley-db"
+  depends_on "libprelude"
+  depends_on "libtirpc"
+  depends_on :linux
+
+  skip_clean :la, "etc"
+
+  def install
+    args = %W[
+      --disable-debug
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --disable-selinux
+      --prefix=#{prefix}
+      --includedir=#{include}/security
+      --oldincludedir=#{include}
+      --enable-securedir=#{lib}/security
+      --sysconfdir=#{etc}
+      --with-xml-catalog=#{etc}/xml/catalog
+      --with-libprelude-prefix=#{Formula["libprelude"].opt_prefix}
+    ]
+
+    system "./configure", *args
+    system "make"
+    system "make", "install"
+  end
+
+  def post_install
+    chmod "u=rwxs,g=rx,o=rx", "#{sbin}/unix_chkpwd"
```

Comment:
What's this doing?

Reply:
@sjackman and @maxim-belkin worked on this in https://github.com/Homebrew/linuxbrew-core/pull/1186 and might know.

Reply:
https://github.com/Homebrew/linuxbrew-core/pull/1186#discussion_r89176752

@maxim-belkin What's the reason for setting the sticky user bit?

Reply:
From the linux-pam LFS [page](http://www.linuxfromscratch.org/blfs/view/svn/postlfs/linux-pam.html):

> The unix_chkpwd helper program must be setuid so that non-root processes can access the shadow file.

In our case, this action has net zero positive effect because Homebrew doesn't use root privileges nor it has an apparatus to temporarily request those. I think we should consider adding such an apparatus to temporarily request root permissions to modify things we need but until then this line can be safely deleted.

Reply:
I removed the post install step.